docs/HANDLED_RULES.md

```diff
@@ -141,6 +141,42 @@ Example:
 
 -----
 
+#### JEE applications should not "getClassLoader" ([Sonar Rule 3032](https://rules.sonarsource.com/java/RSPEC-3032))
+
+Using the standard getClassLoader() may not return the right class loader in Java Enterprise Edition context, i.e. the code use any of these [javax packages](https://docs.oracle.com/javaee/6/api/overview-summary.html). Instead, go through the currentThread. In particular, this processor will transform the code in `.java` files if they appear to use the `javax` package.
```

Comment:
"i.e. the code use any of these [javax packages](https://docs.oracle.com/javaee/6/api/overview-summary.html)" -> the connection of this sentence with the previous one is strange, there is a gap. Try to be clearer.

"these [javax packages]" -> "[these javax packages]"

"Instead, go through the currentThread." -> this sentence is not good. It seems you are asking our users to do something. Replace this sentence by "The calls to `getClassLoader()`, such as `this.getClass().getClassLoader()` and `Dummy.class.getClassLoader()`, are replaced by `Thread.currentThread().getContextClassLoader()`.

"In particular, this processor will transform the code in `.java` files if they appear to use the `javax` package." -> why if they "appear"?

General note for writing this description here: try to be succinct, and to say what exactly is changed by the processor. No much details on technical aspects are needed.

Reply:
Thanks for the input :) . Fixed.